### PR TITLE
docs: add documentation on the gluu specific services required for Admin UI and FAQ #1315

### LINKED ISSUE #1315: docs(admin-ui): add documentation on the gluu specific services required for Admin UI and FAQ
Add documentation on the gluu-specific services required for Admin UI.

### PR DESCRIPTION
**Pull request by duttarnab**
closes #1315

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -138,6 +138,7 @@ nav:
           - 'FIDO': 'admin/admin-ui/fido-menu.md'
           - 'SMTP': 'admin/admin-ui/smtp-menu.md'
           - 'Users': 'admin/admin-ui/userMgmt-menu.md'
+        - 'FAQ & Troubleshooting': 'admin/admin-ui/faq.md'
```
Comment (ossdhaval) [inline mkdocs.yml:141]: Hi @duttarnab 

Changes at this line are fine But check line 132. 
It is reinserting the problematic configuration. Maybe this branch was created before we fixed the issue. Can merge changes from `main` or apply your changes on a fresh branch and create a new PR?

File: docs/admin/admin-ui/faq.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+If a user encounters the above error when visiting the Admin UI URL, it indicates that the Admin UI is not properly installed. Please verify whether the Admin UI build is located at `/var/www/html/admin`. If the build is not present at this location, Janssen displays this error.
+
+####  Admin UI backend is down
+
+```
+Error Code: 503
+Gluu Flex Admin UI is not getting any response from the backend (Jans Config Api).
+```
+
+Gluu Flex Admin UI facilitates interaction with the Jans Auth Server through a REST API layer, [Jans Config API](https://docs.jans.io/v1.0.16/contribute/implementation-design/jans-config-api/). This error prompts administrators to perform a series of troubleshooting steps.
```
Comment (ossdhaval) [inline docs/admin/admin-ui/faq.md:23]: You should use [version placeholder](https://docs.jans.io/head/CONTRIBUTING/#referencing-janssen-project-release-in-documents) when using an full URL.

File: docs/admin/admin-ui/faq.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# Frequently Asked Questions (FAQ)
+
+
+### Why is the Gluu Flex Admin UI displaying the following error messages after the Flex VM installation?
+
+
+#### The requested page not found
+
+```
```
Comment (ossdhaval) [inline docs/admin/admin-ui/faq.md:9]: add ` ```text ` here and other appropriate places.

**Review by ossdhaval — CHANGES_REQUESTED** [review]
A few changes requested in the review comments.

File: mkdocs.yml (hunk 1 above)
Comment (duttarnab) [reply mkdocs.yml:142]: done

File: docs/admin/admin-ui/faq.md (hunk 2 above)
Comment (duttarnab) [reply docs/admin/admin-ui/faq.md]: done

File: docs/admin/admin-ui/faq.md (hunk 3 above)
Comment (duttarnab) [reply docs/admin/admin-ui/faq.md]: done

**Pull request merged**